### PR TITLE
[amqplib] Introduce libdef for amqplib@0.9.1

### PR DESCRIPTION
**Pull request by ahem**
Add libdef for https://github.com/squaremo/amqp.node

### CODE REVIEW

File: definitions/npm/amqplib_v0.5.x/flow_v0.60.1-/amqplib_v0.5.x.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
+// @flow
+
+type Msg = {
```
Comment (AndrewSouthpaw) [inline definitions/npm/amqplib_v0.5.x/flow_v0.60.1-/amqplib_v0.5.x.js]: Put these inside module to not pollute the global namespace.

File: definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+/* @flow */
+
+import type { Message } from 'amqplib/callback_api';
+
+var q = 'tasks';
+
+// ==== promise style =============================================
```
Comment (AndrewSouthpaw) [inline definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js]: Use `describe/it` blocks for better organization

File: definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        consumer_cb(conn);
+        publisher_cb(conn);
+    });
+
+
+
+
+
+
+
```
Comment (AndrewSouthpaw) [inline definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js]: delete extra carriage returns

File: definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
+/* @flow */
```
Comment (AndrewSouthpaw) [inline definitions/npm/amqplib_v0.5.x/test_amqplib_v0.5.x.js]: Put inside version flow directory.

**Comment by ahem** [comment]
@AndrewSouthpaw Ok! I think I addressed all your points now :-)

File: definitions/npm/amqplib_v0.5.x/flow_v0.60.1-/amqplib_v0.5.x.js [hunk 5]
```diff
@@ -0,0 +1,286 @@
+// @flow
+
+declare module 'amqplib' {
+    declare export type AMQPError = any;
```
Comment (AndrewSouthpaw) [inline definitions/npm/amqplib_v0.5.x/flow_v0.60.1-/amqplib_v0.5.x.js]: Indentation should be 2 spaces IIRC. Did prettier not run on your machine? It should've been autoformatted.

**Comment by ahem** [comment]
As far as I can see, the `.eslintrc` file in `definitions/` doesn't include Prettier, and also it doesn't specify any indentation level, but there was an `.editorconfig` file that my editor hadn't picked up. Also, I could have used my eyes and mimicked the rest of the code :-)

...anyway, I fixed the indentation!

**Comment by AndrewSouthpaw** [comment]
Thanks!

**Comment by AndrewSouthpaw** [comment]
Not sure about prettier, maybe we removed that... ¯\_(ツ)_/¯

**Pull request merged**